README.md

```diff
@@ -169,6 +169,7 @@ participants:
 
   # A list of optional extra params that will be passed to the CL to run separate Beacon and validator nodes
   # Only possible for nimbus or teku
+  # Please note that in order to get it to work with Nimbus, you have to use `ethpandaops/nimbus:unstable` as the image (default upstream image does not yet support this out of the box)
```

Comment:
is this still true?

Reply:
Yeah I have just added this comment.

Reply:
See the failed test: https://app.circleci.com/pipelines/github/kurtosis-tech/ethereum-package/2514/workflows/fb2ae4fd-7f44-4e2c-b9e4-a293af6f0003/jobs/10422